db/migrate/20210811211423_add_devise_to_users.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+class AddDeviseToUsers < ActiveRecord::Migration[6.1]
+  def self.up
+    change_table :users do |t|
```

Comment:
Great. This adds the fields required for `:database_authenticatable`, but keep in mind your devise model definition looks like this:

```
devise :database_authenticatable, :registerable,
       :recoverable, :rememberable, :validatable
```

This means you still need to add more fields. Check out [this link](https://github.com/heartcombo/devise/wiki/How-To:-Change-an-already-existing-table-to-add-devise-required-columns) provided by @jclopezdev to add the missing ones.

Reply:
You also need to change `password` to `encrypted_password`.